hail/Makefile

```diff
@@ -1,5 +1,9 @@
 .PHONY: shadowJar jars clean
 
+include env_var.mk
```

Comment:
this sets the default target to `clean-env`. I think we want a `.DEFAULT_GOAL := shadowJar` above this.

Reply:
releaseJar versus shadowJar are built by the same target. `shadowJar`.